src/components/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string | undefined
+}
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId }) => {
```

Comment:
you can use useParams here to get selected tab from route 

```suggestion
interface Props {
  tabs: Tab[],
}
export const Tabs: React.FC<Props> = ({ tabs }) => {
```